lib/index.js

```diff
@@ -82,7 +85,17 @@ function logFacesAppender(config) {
     Object.keys(event.context).forEach((key) => {
       lfsEvent[`p_${key}`] = event.context[key];
     });
-
+    if (config.eventLayout) {
+      lfsEvent = config.eventLayout(event, { ...lfsEvent });
+      if (!lfsEvent) {
```

Comment:
How about the case whereby an empty Object `{}` is returned?
That would still be a valid `LogFacesEvent` according to the typings.

You might also want to include defensive type checks (for non-typescript users) in the event that `eventLayout()` returns wrong types such as `Array[]`. It should be defensively caught and log as error before a hard `return`.